### ISSUE TITLE
Exception thrown on HTC one using web audio plugin.

### ISSUE THREAD

**Issue by peteward44** [CreateJS/SoundJS#83]
I have a HTC one running Android 4.2 with the native browser (it reports it's user agent as safari).

By default, a sound plays fine the first time it is played, but on the second time it throws an exception ("invalid index parameter"). I have traced where it is being thrown from and it is coming from line 918 in WebAudioPlugin.js. It appears to be generated by the WebAudio disconnect function. Here this.panNode is being passed as the first parameter but the WebAudio documentation states it should be an integer index. I have no idea why it only occurs on this device, as everything else works fine.

So, in short, should line 918 of WebAudioPlugin.js be audioNode.disconnect(0); instead?
